packages/langium/src/workspace/configuration.ts

```diff
@@ -174,7 +174,11 @@ export class DefaultConfigurationProvider implements ConfigurationProvider {
         return `${languageId}`;
     }
 
-    get onConfigurationSectionUpdate(): Event<ConfigurationSectionUpdate> {
+    onConfigurationSectionUpdate(callback: ConfigurationSectionUpdateListener): Disposable {
+        return this.onConfigurationSectionUpdateEmitter.event(callback);
+    }
+
+    protected get onConfigurationSectionUpdateEvent(): Event<ConfigurationSectionUpdate> {
```

Comment:
What's the reason for splitting this in two getters?

Reply:
Semantics - reverted to original. 👍